.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  env: {
+    commonjs: true,
+    es2021: true,
+    node: true,
+  },
+  extends: [
+    'airbnb-base',
+  ],
+  parserOptions: {
+    ecmaVersion: 12,
+  },
+  rules: {
```

Comment:
Just a suggestion. I personally prefer 4 spaces indentation so you can set that as a rule here:

```
indent: ["error", 4],
```